ci-operator/config/open-cluster-management/must-gather/open-cluster-management-must-gather-main.yaml

```diff
@@ -0,0 +1,30 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
```

Comment:
`cli` is not a suitable build root, it does not contain `git` (see https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image)

Reply:
Thanks for the tip! Is there a recommended tag?